vm/src/stdlib/os.rs

```diff
@@ -128,30 +121,15 @@ pub fn os_open(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
     };
     let fname = &make_path(vm, name, &dir_fd).value;
 
-    let flags = FileCreationFlags::from_bits(objint::get_value(flags).to_u32().unwrap())
-        .ok_or_else(|| vm.new_value_error("Unsupported flag".to_string()))?;
-
-    let mut options = &mut OpenOptions::new();
+    let mut options = OpenOptions::new();
 
-    if flags.contains(FileCreationFlags::O_WRONLY) {
-        options = options.write(true);
-    } else if flags.contains(FileCreationFlags::O_RDWR) {
-        options = options.read(true).write(true);
+    if cfg!(unix) {
+        let flags = objint::get_value(flags).to_i32().unwrap();
+        options.custom_flags(flags);
     } else {
```

Comment:
![image](https://user-images.githubusercontent.com/16250688/63209969-ac90ec00-c11a-11e9-9884-c4761007b6a3.png)


![image](https://user-images.githubusercontent.com/16250688/63209975-b3b7fa00-c11a-11e9-8ea1-81b44ad77567.png)

![image](https://user-images.githubusercontent.com/16250688/63209981-bfa3bc00-c11a-11e9-92b9-e02a4d64fb4a.png)

I feel strange about the result compilint in ci.